pkg/cmd/attestation/verify/policy.go

```diff
@@ -25,26 +24,71 @@ func expandToGitHubURL(tenant, ownerOrRepo string) string {
 	return fmt.Sprintf("(?i)^https://%s.ghe.com/%s/", tenant, ownerOrRepo)
 }
 
-func buildSANMatcher(opts *Options) (verify.SubjectAlternativeNameMatcher, error) {
+func newEnforcementCriteria(opts *Options) (verification.EnforcementCriteria, error) {
+	c := verification.EnforcementCriteria{}
+
+	// Set SANRegex using either the opts.SignerRepo or opts.SignerWorkflow values
```

Comment:
This section of setting SAN and/or SANRegex is using logic originally from https://github.com/cli/cli/blob/trunk/pkg/cmd/attestation/verify/policy.go#L28-L44